utilities/email_helper.py

```
@@ -0,0 +1,283 @@
+        valid_emails = [email.strip() for email in emails if email and email.strip()]
+        
+        # Update league
+        await db.execute(
+            update(League)
+            .where(League.id == league_id)
+            .values(league_member_emails=json.dumps(valid_emails))
+        )
+        await db.commit()
+        
+        print(f"Set {len(valid_emails)} emails for league {league_id}")
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)